### PR TITLE
Console api metric

### PR DESCRIPTION
**Pull request by xinlili-statsig**
Update metric creation api

### CODE REVIEW

File: src/components/Rapidoc.js
Comment (jacob-statsig) [inline src/components/Rapidoc.js:1]: Didn't think we'd need a linter for Docs but here we are

**Comment by jasonwzm** [comment]
🚀 Deployed on https://64dd3dca9e19d107840832f5--cozy-fox-0defec.netlify.app

**Comment by jacob-statsig** [comment]
Looks like one of the examples isn't working 
image

**Comment by jacob-statsig** [comment]
and the other examples gives this
image

**Comment by jacob-statsig** [comment]
under [POST /gates/{gate_id}](https://docs.statsig.com/console-api/gates#post-/gates/-gate_id-) we have an enum value we should use. We are getting some errors for enums 
Gates status:
image
VS
Your Metrics Creation:
image

**Comment by xinlili-statsig** [comment]
enum issue fixed
remove one example
sum example should be fixed

**Pull request merged**